dist/lightbox-dialog/ds6/lightbox-dialog.css

```diff
@@ -192,24 +203,24 @@
   }
 }
 @media (min-width: 601px) {
-  .lightbox-dialog__header {
+  .lightbox-dialog__window:not(.lightbox-dialog__window--alert):not(.lightbox-dialog__window--confirm) .lightbox-dialog__header {
```

Comment:
Hmm, I think some of these selectors aren't great, and a bit of a code smell. I believe that's why I avoided header and footer structure in confirm and alert altogether, because the selectors were definitely simpler before. The only downside was it lacked the consistency and structure of course. Is the main difference the margins? If so I wonder if we could replace `.lightbox-dialog__window--alert` and `.lightbox-dialog__window--confirm` with something more generic like `.lightbox-dialog__window--compact` or `.lightbox-dialog__window--centered`. Have a think what we could do to clean this up a bit, and I'll do the same.

Reply:
We could possibly even split out into their own modules: `.confirm-dialog` and `.alert-dialog` and use mixins. Although that's slightly odd because they are also lightbox-dialogs...

Reply:
I think one of the solutions is not to use `lightbox-dialog__window` for alert or confirm dialogs. This way we can have all the logic we want in `window`. Adding new selectors makes it more complex imo. 
I don't think having their own modules also helps. 
The main difference is some margins but also some other things like button positions. Having the same structure is best imo and helps ebayui since we don't have to replicate header logic in other places.

Reply:
I updated the PR to use compact. Basically it won't use `lightbox-dialog__window` anymore but `lightbox-dialog__window--compact`. 
If this looks good, I will update the ebayui version of it too.

Reply:
@agiga It is invalid BEM to have a modifier in the HTML (`lightbox-dialog__window--compact`) without also a base class (`lightbox-dialog__window`), so we'll have to rethink this.

Reply:
I think a better idea would be to do another window type like: `lightbox-dialog__compact-window`. This would work and not require window.

Reply:
Yep, that's better. I'll take some time to review the rest...